### ISSUE TITLE
Outdated actions for forge-bot

### ISSUE THREAD

**Issue by Jonathing** [MinecraftForge/SharedActions#9]
Several bot-related actions are out of date with old or dead links, incorrect labels, and unnecessary behaviors. The most egregious one is how many repositories' actions contain a dead Discord link.

This issue is for tracking purposes. I hope to work on this soon to remedy the problem.

This issue applies across the entire organization except for repositories that use their own actions (like [MinecraftForge](https://github.com/MinecraftForge/MinecraftForge)).

**Comment by LexManos** [comment]
Not quite sure what you mean by dead discord link, there are a few repos with commented things linking to old discord.gg links but everything in SharedActions either uses a config value or is updated to the domain we control (discord.minecraftforge.net)
But yes, Actionable/Shared actions needs cleanup as it has too many things that were never useful.

**Comment by Jonathing** [comment]
Yes, this is mostly the "holy shit, this is 1,000 years expired milk, we need to clean this up" kind of issue. If you could assign me to it (and/or give me triage access to this repo) that'd be great.
